.github/workflows/check.yaml

```diff
@@ -1,6 +1,8 @@
 name: Tests
 
 on:
+  push:
+    branches: [ main ]
```

Comment:
This isn't needed - the check workflow is called by the release workflow on push to main already: https://github.com/canonical/charmed-openstack-upgrader/blob/db6c4cdc51d0b12ccc40a65fed681752b9a8cf0d/.github/workflows/release.yaml#L9-L10